SAMv1.tex

```diff
@@ -296,6 +304,7 @@ \subsection{The alignment section: mandatory fields}
     0x40 and 0x80 are unset, the index of the read in the template
     is unknown. This may happen for a non-linear template or the index
     is lost in data processing.
+  \item Bit 0x20 defines the next segment as per the definition in RNEXT and PNEXT.
```

Comment:
I'm not sure what this means. If the next segment is the third segment in a 5 segment template, what does having 0x20 set actually mean? Alignments for segments 2 and 3 are consistent? 3 and 4? All segment alignment are consistent with having no (large) SVs wrt the set of reference alignment?